app/uk/gov/hmrc/transitmovementsrouter/connectors/PersistenceConnector.scala

```diff
@@ -61,7 +60,7 @@ trait PersistenceConnector {
 }
 
 @Singleton
-class PersistenceConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig: AppConfig) extends PersistenceConnector with Logging {
+class PersistenceConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig: AppConfig) extends PersistenceConnector {
```

Comment:
No change to suggest but @drnaylor-oc do you think we should consider our rules for logging regarding connectors? Feels weird to log nothing.

Reply:
Yeah, we should really. Maybe have something like we do in the EIS router for P4 in all our connectors, so we can pin-point problems in the connectors quickly, perhaps?

Reply:
Lets have that discussion outside the PR!